OpenTabletDriver.Configurations/Configurations/Huion/Kamvas Pro 19 (4k).json

```
@@ -0,0 +1,42 @@
+{
+    "Name": "Huion Tablet_GT1902",
+    "Specifications": {
+        "Digitizer": {
+            "Width": 409,
+            "Height": 230,
+            "MaxX": 81792,
+            "MaxY": 46006
+        },
+        "Pen": {
+            "MaxPressure": 16384,
+            "ButtonCount": 2
+        },
+        "AuxiliaryButtons": null,
+        "MouseButtons": null,
+        "Touch": {
+            "Width": 409,
+            "Height": 230,
+            "MaxX": 163843,
+            "MaxY": 9600
+        }
+    },
+    "DigitizerIdentifiers": [
+        {
+            "VendorID": 9580,
+            "ProductID": 107,
+            "InputReportLength": null,
```

You are letting every interface of this tablet match which is the likely cause of the need to replug.

Try this, and if this doesn't work try `14`, there should be absolutely no reason to use `null`, all interfaces do not work.
```suggestion
            "InputReportLength": 28,
```